packages/api/package.json

```diff
@@ -54,12 +55,12 @@
     "prompts": "^2.4.2",
     "semver": "^7.3.8",
     "ssri": "^10.0.1",
-    "ts-morph": "^17.0.1",
+    "ts-morph": "^20.0.0",
+    "tsup": "^7.2.0",
```

Comment:
could we remove this as a dep since it's being installed in the codegen'd directory now?